docs/notes/workouts.json.md

```diff
@@ -0,0 +1,10 @@
+---
+id: dpeysudtg6of05oz7odgd4a
+title: Json
+desc: ''
+updated: 1678569579309
+created: 1678569523359
+---
+**Questions**
+
+1. What does the boolean in the Array items do and why are some items set to false and others to true?
```

Comment:
This makes for a great issue where we can discuss where this should be heading in the future - apart from it, I think the whole `docs`/`dendron` changes were accidentally commited to the branch?

Reply:
Thanks for the feedback. I agree to add the new routines to the bottom of the list. I'll make the changes and submit them for your review. Thanks again for your help